codegen/module.go

```diff
@@ -284,18 +298,6 @@ func (system *ModuleSystem) ResolveModules(
 		if err != nil {
 			return nil, err
 		}
-
-		// Resolve dependencies again in case two instances
```

Comment:
I swear we have to double resolve because of peer deps anyway... This might break peer deps. Needs tests to verify.